src/oscar/utils/models.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+from django.conf import settings
+from django.utils import timezone
+
+
+def get_image_upload_path(instance, filename):
+    return os.path.join(timezone.now().strftime(settings.OSCAR_IMAGE_FOLDER), filename)
```

Comment:
Suggest we use `datetime.datetime.now()` to be consistent with [what Django does](https://github.com/django/django/blob/aed89adad54a977829c4f180b036033e031ebcc7/django/db/models/fields/files.py#L305) - there is a difference in terms of timezone-awareness.

Reply:
We should also use `posixpath.join` as Django does.